Content.Server/Chat/V2/Messages.cs

```diff
@@ -1,94 +1,125 @@
-﻿using Content.Shared.Chat.Prototypes;
-using Content.Shared.Chat.V2;
+﻿using Content.Shared.Chat.V2;
```

Comment:
There's a bunch of changes in this file; the basics are compression down to the few relevant pathways (i.e. how messages  get sent)